### PR TITLE
ruRU locales

### PR DESCRIPTION
**Pull request by DarkQuantum**
Add ruRU locales, edit main file.

### CODE REVIEW

**Comment by DarkQuantum** [comment]
ruRU.lua must be encoded in UTF-8

**Comment by funkydude** [comment]
Missing:

```
LegionInvasionTimer.toc ## Notes
Options\Locales\ruRU.lua
Options\LegionInvasionTimer_Options.toc ## Notes
```

**Comment by DarkQuantum** [comment]
Done :)

Options\Locales\ruRU.lua in UTF-8

File: Locales/ruRU.lua [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
 local _, mod = ...
 local L = mod.L
 
 -- Core
-L.invasionsAvailable = "|T236292:15:15:0:0:64:64:4:60:4:60|t New invasions available!"
-L.tooltipClick = "|cffeda55fClick|r to drag and move."
-L.tooltipClickOptions = "|cffeda55fRight-Click|r to open options."
-L.searching = "Searching..."
+L.invasionsAvailable = "|T236292:15:15:0:0:64:64:4:60:4:60|t Новое вторжение доступно!"
+L.tooltipClick = "|cffeda55fClick|r преподнимите и переместите."
```
Comment (funkydude) [inline Locales/ruRU.lua]: Missing translation for `Click` and `Right-Click`

**Comment by funkydude** [comment]
You'll need to merge the latest version since there's conflicts now.

**Pull request closed without merging**